config/MAINTAINERS.json

```diff
@@ -0,0 +1,616 @@
+[
```

Comment:
please add folks from https://github.com/asyncapi/dotnet-rabbitmq-template/blob/master/CODEOWNERS that are maintainers, but not TSC members

Reply:
I have added `@connil` and set the value of its `isTscMember` key to `false`. It is not getting rendered.